src/dex-ui/pages/dao/TokenTransactionDetails/TokenTransactionDetails.tsx

```diff
@@ -22,6 +30,8 @@ import { TransactionStatusAsTagVariant } from "../constants";
 
 export function TokenTransactionDetails() {
   const { accountId: daoAccountId = "", transactionHash = "" } = useParams();
+  const { wallet } = useDexContext(({ wallet }) => ({ wallet }));
+  const connectedWalletId = wallet.getSigner().getAccountId().toString();
```

Comment:
Getting wallet id from `wallet.getSigner().getAccountId().toString()` gives error sometimes when the page reloads I fixed the same issue in proposal details page when user refreshes the proposal detail page so I changed the approach and picked wallet id from `savedPairingData`. just for safety purpose just check your case as well.

Reply:
Nice catch. The page failed on refresh with the following error:
<img width="950" alt="Screen Shot 2023-05-15 at 2 07 47 PM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/97d31367-8339-4798-9c66-ec870b5e3a17">
I've updated the code to source the account id from `savedPairingData`.